README

```diff
@@ -0,0 +1,15 @@
+# MyRental Express Server
```

Comment:
If there is anything that would be helpful for everyone else to know, now would be a good time to add it.